source/stylist/configuration.py

```diff
@@ -124,7 +124,28 @@ def get_style(self, name: str) -> Sequence[str]:
             if len(rules.strip()) == 0:
                 raise StylistException(f"Style {key} contains no rules")
             else:
-                return rules.split(',')
+                rule_list: List[str] = []
+                state = 'rule'
```

Comment:
You could perhaps store `'rule'` and `'args'` somewhere rather than have them as literals?  On the off-chance the logic here ever expands and you want to rename either of them